src/pages/splash.tsx

```diff
@@ -211,12 +211,14 @@ export const SplashPage: React.FunctionComponent<{
 
   const [playYoutubeTrailer, setPlayYoutubeTrailer] = useState(false);
   return (
-    <>
-      <Head>
-        <meta name="theme-color" content="#1C0D28" />
-        <meta name="viewport" content="width=device-width, initial-scale=1" />
-        <title>Biomes — Join the community shaping a new world</title>
-      </Head>
+    <StaticPage
```

Comment:
Can you check the root page too? I think it is index.tsx — I believe it does something different than splash

Reply:
Should be fine, but it's been updated!
```ts
<RootErrorBoundary>
    <BiomesHeadTag description="Biomes is an open source sandbox MMORPG built for the web using web technologies" />
    <SplashPage
      onLogin={() => {
        window.location.href = "/at";
      }}
    />
  </RootErrorBoundary>
  ```